DragAndDrop/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,15 @@
         <activity
             android:name="com.microsoft.device.display.samples.draganddrop.DragAndDropActivity"
             android:label="@string/drag_and_drop_app_name" />
+        <provider
+            android:name="androidx.core.content.FileProvider"
+            android:authorities="com.microsoft.device.display.samples.duosamples.images"
```

Comment:
Probably better to use an authority that matches sample package name? e.g. instead of `com.microsoft.device.display.samples.duosamples.images` use `com.microsoft.device.display.samples.images`

Reply:
The package name for the app is "com.microsoft.device.display.samples.duosamples"